pkgs/development/python-modules/jaxtyping/default.nix

```
@@ -39,17 +49,23 @@ let
     nativeCheckInputs = [
       cloudpickle
       equinox
+      ipython
       jax
       jaxlib
       pytestCheckHook
+      tensorflow
       torch
     ];
 
     doCheck = false;
 
     # Enable tests via passthru to avoid cyclic dependency with equinox.
     passthru.tests = {
-      check = self.overridePythonAttrs { doCheck = true; };
+      check = self.overridePythonAttrs {
+        # We disable tests because they complain about the version of typeguard being too new.
+        doCheck = true;
```

`doCheck = false`?

---

Yes, the problem is that the tests refuse to run because typeguard is too recent.
I could remove the whole `check = self.overridePythonAttrs {` block but I wanted to keep it for us to re-enable tests later.
Please, tell me if you prefer to simply get rid of it for now.

---

My apologies. I had to explain it more.
No need to remove it, just change `doCheck = false`.
```suggestion
        doCheck = false;
```

---

Ok done !